src/gleam_community/codec.gleam

```diff
@@ -1,17 +1,17 @@
 // IMPORTS ---------------------------------------------------------------------
 
+import gleam/dict.{type Dict}
 import gleam/dynamic.{
   type DecodeError as DynamicError, type Dynamic, DecodeError as DynamicError,
 }
 import gleam/function
+import gleam/int
 import gleam/json.{type DecodeError as JsonError, type Json}
 import gleam/list
-import gleam/dict.{type Dict}
 import gleam/option.{type Option}
 import gleam/pair
 import gleam/result
-import gleam/string_builder.{type StringBuilder}
-import gleam/int
+import gleam/string_tree.{type StringTree}
```

Comment:
using `gleam/string_tree` means we need to bump our gleam_stdlib constraint to at least 0.42.0